plugins/tiddlywiki/prosemirror/slash-menu/slash-menu-ui.js

```diff
@@ -177,25 +177,25 @@ SlashMenuUI.prototype.createMenuItem = function(element, state) {
 
 SlashMenuUI.prototype.getIconForElement = function(element) {
 	// Simple text icons for now - can be enhanced with SVG later
```

Comment:
Will they ever be SVG icons "later" -- At least add a TODO the comment, so anyone may find it later ;)